conf/conf.go

```diff
@@ -436,7 +436,9 @@ func isProbe(t reflect.Type) bool {
 // AllProbers return all probers
 func (conf *Conf) AllProbers() []probe.Prober {
 	log.Debugf("--------- Process the probers settings ---------")
-	return allProbersHelper(*conf)
+	all := allProbersHelper(*conf)
+	MergeConstLabels(all)
```

Comment:
`AllProbers` may be called more than once (even it is not currently, the name suggest it could be), which result in `MergeConstLabels` be called more than once, this is not necessary and may cause a bug in the future.

propose moving `MergeConstLabels` to somewhere that will only be called once.

Reply:
updated.